src/view/background/LineBackgroundType.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * Xournal++
+ *
+ * The cursor types for selections
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+enum LineBackgroundType
+{
+	LINE_BACKGROUND_LINED,
```

Comment:
Same as above, why not PascalCase for Enums? Also could make this an `enum class`